internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -776,20 +786,22 @@ func (rmq *rocksmq) updateAckedInfo(topicName, groupName string, newID UniqueID,
 	lock.Lock()
 	defer lock.Unlock()
 
+	lastID := ids[len(ids)-1]
+
 	fixedBeginIDKey, err := constructKey(BeginIDTitle, topicName)
 	if err != nil {
 		return err
 	}
 	// Update begin_id for the consumer_group
 	beginIDKey := fixedBeginIDKey + "/" + groupName
-	err = rmq.kv.Save(beginIDKey, strconv.FormatInt(newID, 10))
+	err = rmq.kv.Save(beginIDKey, strconv.FormatInt(lastID, 10))
 	if err != nil {
 		return err
 	}
 
 	// Update begin_id for topic
 	if vals, ok := rmq.consumers.Load(topicName); ok {
-		var minBeginID int64 = -1
+		var minBeginID int64 = math.MaxInt64
 		for _, v := range vals.([]*Consumer) {
```

Comment:
Could vals be empty?

Reply:
Couldn't.

Reply:
@czs007 there must be a consumer associated with this `topicName` to call this function.